src/Profiling/Profiler.php

```diff
@@ -65,13 +73,23 @@ public function getProfile(): ?Profile
 
     private function initProfiler(): void
     {
-        if (\extension_loaded('excimer') && \PHP_VERSION_ID >= 70300) {
-            $this->profiler = new \ExcimerProfiler();
-            $this->profile->setStartTimeStamp(microtime(true));
+        if (\PHP_VERSION_ID < 70300) {
```

Comment:
Hmm, this should also work with PHP 7.1 https://pecl.php.net/package/excimer

Reply:
Hmm, yeah we don't even support that version in the SDK (7.2+ only). Maybe we remove the check? Can't find why we even added it originally?

Reply:
Yep, that was it!